### PR TITLE
PHOENIX-5997 Phoenix Explain Plan for Deletes does not clearly differentiate between server side and client side paths.

### PR DESCRIPTION
**Pull request by dbwong**
https://issues.apache.org/jira/browse/PHOENIX-5997

### CODE REVIEW

**Comment by dbwong** [comment]
Whoops just noticed the formating is wrong let me fix that real quick.

File: phoenix-core/src/test/java/org/apache/phoenix/query/ExplainPlanTextTest.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
+package org.apache.phoenix.query;
```
Comment (gjacoby126) [inline phoenix-core/src/test/java/org/apache/phoenix/query/ExplainPlanTextTest.java:1]: Missing an Apache License

**Comment by dbwong** [comment]
@gjacoby126 added header.

**Comment by yanxinyi** [comment]
@gjacoby126 Hi Geoffrey, is there anything else other than adding Apache License?

**Comment by gjacoby126** [comment]
JIRA already committed, closing PR

**Pull request closed without merging**